tasmota/tasmota_xdrv_driver/xdrv_23_zigbee_7_0_statemachine.ino

```diff
@@ -911,7 +919,7 @@ static const Zigbee_Instruction zb_prog[] PROGMEM = {
 
     // add endpoint 0x01 and 0x0B
     ZI_SEND(ZBS_ADD_ENDPOINT1)          ZI_WAIT_RECV(2500, ZBR_ADD_ENDPOINT)
-    ZI_SEND(ZBS_ADD_ENDPOINTB)          ZI_WAIT_RECV(2500, ZBR_ADD_ENDPOINT)
+//    ZI_SEND(ZBS_ADD_ENDPOINTB)          ZI_WAIT_RECV(2500, ZBR_ADD_ENDPOINT)
```

Comment:
Why did you remove endpoint 0x0B? It is needed for some devices (don't remember if it's Philips or Xiaomi) that have this value hardcoded.